templates/typescript/samples/sample-assistant/src/index.ts

```diff
@@ -156,34 +168,41 @@ try {
     // Configure bot services
     const botServices: BotServices = new BotServices(botSettings, telemetryClient);
 
-    const skillContextAccessor: StatePropertyAccessor<SkillContext> = userState.createProperty<SkillContext>(SkillContext.name);
     const userProfileStateAccesor: StatePropertyAccessor<IUserProfileState> = userState.createProperty<IUserProfileState>('IUserProfileState');
-    const onboardingDialog: OnboardingDialog = new OnboardingDialog(userProfileStateAccesor, botServices , localeTemplateEngine, telemetryClient);
+    const onboardingDialog: OnboardingDialog = new OnboardingDialog(userProfileStateAccesor, botServices, localeTemplateEngine, telemetryClient);
     const switchSkillDialog: SwitchSkillDialog = new SwitchSkillDialog(conversationState);
     const previousResponseAccesor: StatePropertyAccessor<Partial<Activity>[]> =
-    userState.createProperty<Partial<Activity>[]>('Activity');
+        userState.createProperty<Partial<Activity>[]>('Activity');
 
-    // Register skill dialogs
-    const skillDialogs: SkillDialog[] = skills.map((skill: ISkillManifest): SkillDialog => {
-        const authDialog: MultiProviderAuthDialog|undefined = buildAuthDialog(skill, botSettings, appCredentials);
-        const credentials: MicrosoftAppCredentialsEx = new MicrosoftAppCredentialsEx(
-            botSettings.microsoftAppId || '',
-            botSettings.microsoftAppPassword || '',
-            skill.msaAppId);
 
-        return new SkillDialog(skill, credentials, telemetryClient, skillContextAccessor, authDialog);
-    });
-    
+    let skillHttpClient: SkillHttpClient = new SkillHttpClient(
+        credentialProvider,
+        new SkillConversationIdFactory(storage),
+        undefined
+    );
```

Comment:
```suggestion
    let skillHttpClient: SkillHttpClient = new SkillHttpClient(
        credentialProvider,
        new SkillConversationIdFactory(storage)
    );
```

The last parameter could be optional, we can remove that undefined parameter